mpv-root/installer/updater.ps1

```diff
@@ -425,7 +482,14 @@ function Upgrade-Ytplugin {
         }
         else {
             Write-Host "Newer" (Get-Item $yt).BaseName "build available" -ForegroundColor Green
-            & $yt --update
+            $ytdl = Check-GetYTDL
+            if ($ytdl -eq 'ytdlp') {
```

Comment:
If someone have a old ytdl/ytdlp and not have `getytdl` in setting.xml, `Check-GetYTDL` will stil ask "Download ytdlp or youtubedl?".
`Check-GetYTDL` should first check if yt-dl or yt-dlp already exists and set them to `getytdl`.
Similarly it seems appropriate to consider a special case where yt-dl already exists, but manually modified `getytdl` to `ytdlp`